spring-boot-dixit/src/main/java/tw/wally/dixit/DixitApplicationRunner.java

```diff
@@ -0,0 +1,26 @@
+package tw.wally.dixit;
+
+import lombok.AllArgsConstructor;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.stereotype.Component;
+import tw.wally.dixit.clients.LobbyServiceDriver;
+
+import static tw.wally.dixit.model.Dixit.MAX_NUMBER_OF_PLAYERS;
+import static tw.wally.dixit.model.Dixit.MIN_NUMBER_OF_PLAYERS;
+
+/**
+ * @author - wally55077@gmail.com
+ */
+@Component
+@AllArgsConstructor
+public class DixitApplicationRunner implements ApplicationRunner {
+    private final LobbyServiceDriver lobbyServiceDriver;
+
+    @Override
+    public void run(ApplicationArguments args) {
+        Registration registration = new Registration(MIN_NUMBER_OF_PLAYERS, MAX_NUMBER_OF_PLAYERS);
+        registration.addOption(new Registration.Option("winningGoal", "range", 25, 35, 5));
+        lobbyServiceDriver.registerService(registration);
```

Comment:
Adopt error handling for this line, should you stop the whole application if the registration errs?

Reply:
It will throw a `RuntimeExcetion` to stop the application when the application is started.

Reply:
Got it.